CMakeLists.txt

```diff
@@ -13,13 +13,33 @@ add_library(net STATIC
 target_link_libraries(net
         PUBLIC roboteam_proto)
 
+add_library(worldUtils STATIC
+        include/roboteam_world/filters/KalmanFilter.h
+        include/roboteam_world/filters/WorldFilter.h
+        src/filters/WorldFilter.cpp
+        include/roboteam_world/filters/RobotFilter.h
+        src/filters/RobotFilter.cpp
+        include/roboteam_world/filters/BallFilter.h
+        src/filters/BallFilter.cpp
+        include/roboteam_world/filters/CameraFilter.h
+        src/filters/CameraFilter.cpp
+        src/filters/Scaling.h
+        )
```

Comment:
All header files can be removed, since they are included via the .cpp files

Reply:
This is consistent with the rest of our codebase (and makes porting easier if we ever want to go to another build system.)